canonical-web-and-design/ubuntu.com
# Remove Coming soon and Launching soon on /gcp and /gcp/pro pages

## Done

- Remove "Coming soon" and "Launching soon" on /gcp and /gcp/pro pages
- Changed range from '3-6' to '3-4.5'
- Updated copy docs - [index](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#) and [pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.c6wjbvvp8mcm)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gcp and http://0.0.0.0:8001/gcp/pro
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)


## Issue / Card

Fixes #9509
